CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
+ - Fixed typing in the address bar. ([#3928](https://github.com/brave/browser-laptop/issues/3928))([#3927](https://github.com/brave/browser-laptop/issues/3927))([#3857](https://github.com/brave/browser-laptop/issues/3857))([#3718](https://github.com/brave/browser-laptop/issues/3718))([#3712](https://github.com/brave/browser-laptop/issues/3712))([#3933](https://github.com/brave/browser-laptop/issues/3933))
+ - Fixed Autocomplete for Twitter login. ([#3915](https://github.com/brave/browser-laptop/issues/3915))
+ - Fixed 1Password not filling the form and Brave saving the password. ([#3897](https://github.com/brave/browser-laptop/issues/3897))
+ - Fixed enabling "Web notifications" on Twitter. ([#3579](https://github.com/brave/browser-laptop/issues/3579))
+ - Fixed Brave crash when Dashlane password manager is selected. ([#3909](https://github.com/brave/browser-laptop/issues/3909))
+ - Fixed crash when Linux users would check for updates. ([#3907](https://github.com/brave/browser-laptop/issues/3907))
+ - Fixed Redhat desktop launcher. ([#3905](https://github.com/brave/browser-laptop/issues/3905))
+ - Fixed setting Brave as the default browser under Linux. ([#3776](https://github.com/brave/browser-laptop/issues/3776))
+ - Fixed privacy retention for new tabs opened via context menu actions. ([#3662](https://github.com/brave/browser-laptop/issues/3662))([#3895](https://github.com/brave/browser-laptop/issues/3895))
+ - Fixed a Lastpass create account issue. ([#2936](https://github.com/brave/browser-laptop/issues/2936))
+ - Fixed renderer crash after signing in to https://*.slack.com/home. ([#3841](https://github.com/brave/browser-laptop/issues/3841))
+ - Fixed mixed content pages incorrectly showing as secure in private browsing mode. ([#3793](https://github.com/brave/browser-laptop/issues/3793))([#3795](https://github.com/brave/browser-laptop/issues/3795))
+ - Fixed Mixed-content lock control to re-lock and reload the page. ([#3770](https://github.com/brave/browser-laptop/issues/3770))([#3808](https://github.com/brave/browser-laptop/issues/3808))
+ - Fixed Mac menubar being inaccessible while download is in progress. ([#3823](https://github.com/brave/browser-laptop/issues/3823))
+ - Fixed Cmd+T (New Tab) shortcut becoming disabled while navigation occurs. ([#3782](https://github.com/brave/browser-laptop/issues/3782))
+ - Changed '...' to Unicode ellipsis. ([#3675](https://github.com/brave/browser-laptop/issues/3675))
+ - Updated Electron to 1.3.12 .
```

Comment:
This should be removed, it's only included for upstream upgrades which we haven't done yet.  There will be an issue that describes it explicitly in the milestone when we do.